.changeset/strange-roses-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Client-side router ignores links outside app target
```

Comment:
```suggestion
[breaking] Make client-side router ignore links outside app target
```

Since `target` isn't something that's in the app's config anymore, I don't know whether we ought to use a different word here. Maybe:

```suggestion
[breaking] Make client-side router ignore links outside %sveltekit.body%
```

?